Add changelog for 5.11.0 -> 5.12.0 (#229)
luanti-org/docs.luanti.org: content/about/changelog.md

- `load_mod_* = false` lines are no longer written to `world.mt` (_appgurueu_)

### Script API / Modding

> - glTF models: local transforms no longer affect a skinned mesh. This used to cause scaling issues. (_appgurueu_)

Reviewer: ```suggestion
- glTF models: Local transforms no longer affect a skinned mesh. This bug used to cause wrong rendering of models. (_appgurueu_)
```

scaling issues are one possible kind of issue, all kinds of things can happen

>   - Parent nodes (not bones) should not have transformations.

Reviewer: ```suggestion
  - Tip: To avoid the bug on older Luanti client versions, simply remove the offending local transforms. You can use [glTF validator](https://github.khronos.org/glTF-Validator/) to check whether your models are affected: The `NODE_SKINNED_MESH_LOCAL_TRANSFORMS` warning should not appear.
```

a bit verbose but hopefully more actionable if anyone needs it

> - Fix visual_scale not applying to skeletally animated meshes when used for nodes (_appgurueu_)

Reviewer: ```suggestion
- Fix `visual_scale` not applying to skeletally animated meshes when used for nodes (_appgurueu_)
```

nitpick, i like `inline code` :)
